crates/re_arrow_store/tests/data_store.rs

```diff
@@ -889,29 +897,19 @@ fn gc_impl(store: &mut DataStore) {
         }
         _ = store.to_dataframe(); // simple way of checking that everything is still readable
 
-        // TODO(#1619): bring back garbage collection
-
-        // let row_id_chunks = store.gc(
-        //     GarbageCollectionTarget::DropAtLeastPercentage(1.0 / 3.0),
-        //     Timeline::new("frame_nr", TimeType::Sequence),
-        //     MsgId::name(),
-        // );
+        let stats = DataStoreStats::from_store(store);
 
-        // let row_ids = row_id_chunks
-        //     .iter()
-        //     .flat_map(|chunk| arrow_array_deserialize_iterator::<Option<MsgId>>(&**chunk).unwrap())
-        //     .map(Option::unwrap) // MsgId is always present
-        //     .collect::<ahash::HashSet<_>>();
-
-        // for row_id in &row_ids {
-        //     assert!(store.get_msg_metadata(row_id).is_some());
-        // }
-
-        // store.clear_msg_metadata(&row_ids);
+        let (row_ids, stats_diff) =
+            store.gc(GarbageCollectionTarget::DropAtLeastPercentage(1.0 / 3.0));
+        for row_id in &row_ids {
+            assert!(store.get_msg_metadata(row_id).is_none());
+        }
 
-        // for row_id in &row_ids {
-        //     assert!(store.get_msg_metadata(row_id).is_none());
-        // }
+        assert!(
+            (stats.total_size_bytes as f64 * 0.95 / 3.0) as u64 <= stats_diff.total_size_bytes
+                && stats_diff.total_size_bytes
+                    <= (stats.total_size_bytes as f64 * 1.05 / 3.0) as u64
+        );
```

Comment:
let's make this readable

```suggestion
        let num_bytes_dropped = stats_diff.total_size_bytes as f64;
        assert!(
            (stats.total_size_bytes as f64 * 0.95 / 3.0) <= num_bytes_dropped
                && num_bytes_dropped <= stats.total_size_bytes as f64 * 1.05 / 3.0
        );
```